homeassistant/components/weatherkit/entity.py

```diff
@@ -0,0 +1,33 @@
+"""Base entity for weatherkit."""
+
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import WeatherKitDataUpdateCoordinator
+
+
+class WeatherKitEntity(Entity):
+    """Base entity for all WeatherKit platforms."""
+
+    _base_unique_id: str
```

Comment:
We don't seem to use this?

Reply:
Ah yeah, just a leftover I forgot to remove.